kapitan/inputs/helm/__init__.py

```diff
@@ -18,6 +19,21 @@ def __init__(self, compile_path, search_paths, ref_controller):
         super().__init__("helm", compile_path, search_paths, ref_controller)
         self.helm_values_file = None
         self.helm_params = {}
+        self.lib = self.initialise_binding()
+
+    def initialise_binding(self):
+        """returns the dl_opened library (.so file) if exists, otherwise None"""
+        if platform.system() not in ('Linux', 'Darwin'):  # TODO: later add binding for Mac
+            return None
+        binding_path = os.path.join(os.path.dirname(os.path.abspath(__file__)), "libtemplate.so")
```

Comment:
Add a small comment above this to say what path this builds exactly (where it's pointing too). 
So it's intuitive when changing this in the future for OSX. :)